tests/roots/test-video/mp4_options.rst

```diff
@@ -5,6 +5,7 @@ video
     :alt: You cannot display videos
     :autoplay:
     :nocontrols:
+    :controlslist: nodownload nofullscreen noremoteplayback
```

Comment:
Usually, Sphinx list parameter are set as comma separated values in Sphinx directives, can you adpt your code to fit this unwritten and undocumented standard ? 😄

Reply:
I'll check if `docutils` has a cleaner way to do this and get back in a few days.